### PR TITLE
More Musical Cargo

### PR DESCRIPTION
**Pull request by Endecc**
## About the PR

Adds a whole bunch of Instruments to the Cargo manifest, via bundles organized (roughly) by Type

**Media**

[Options within the Cargo UI](https://i.imgur.com/H8oQ1oP.png)
[Contents of each box (Variety, Brass, String, Woodwind, Keyed/Percussion, Specialty](https://i.imgur.com/5HLLU0A.png)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**

:cl: EnDecc
- add: Added String, Woodwind, Keyed and Percussion, and Specialty Music crates to Cargo's Manifest
- tweak: Renamed 'Big Band' crate to Variety Instrument collection, tweaked ID of Brass crate to easily identify with other Instrument Crate IDs, and its contents.

### CODE REVIEW

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+
+ent-CrateFunInstrumentsWoodwind = Woodwind instrument ensemble crate
+    .desc = If atmos is good at their job, use air to play music with these woodwind instruments! Real wood not guaranteed with every item.
+
+ent-CrateFunInstrumentsKeyedPercussion = Keyed/Percussion instrument ensemble crate
+    .desc = Hit some keys with some sticks or your hands, with this Keyed and Percussion instrument ensemble crate.
+
+ent-CrateFunInstrumentsSpecial = Special instrument collector's crate
+    .desc = Create some noise with this special collection of arguably-instruments! Centcomm is not responsible for any trauma caused by the contents.
+
```
Comment (moonheart08) [inline Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl:21]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
who did this??
upstream isn't supposed to be using entity translation keys but oh well i guess?

File: Resources/Prototypes/Catalog/Cargo/cargo_fun.yml [hunk 2]
```diff
@@ -1,23 +1,63 @@
...
-  product: CrateFunBrass
+  product: CrateFunInstrumentsBrass
   cost: 2500
   category: Fun
   group: market
 
+- type: cargoProduct
+  id: FunInstrumentsString
+  icon:
+    sprite: Objects\Fun\Instruments\bassguitar.rsi
```
Comment (Endecc) [inline Resources/Prototypes/Catalog/Cargo/cargo_fun.yml:24]: My jackass copied the file path from the file explorer, and the slashes are backwards. I've only just realized now. Incredible
```suggestion
    sprite: Objects/Fun/Instruments/bassguitar.rsi
```

File: Resources/Prototypes/Catalog/Cargo/cargo_fun.yml [hunk 3]
```diff
@@ -1,23 +1,63 @@
...
+    state: icon
+  product: CrateFunInstrumentsString
+  cost: 2500
+  category: Fun
+  group: market
+
+- type: cargoProduct
+  id: FunInstrumentsWoodwind
+  icon:
+    sprite: Objects\Fun\Instruments\harmonica.rsi
```
Comment (Endecc) [inline Resources/Prototypes/Catalog/Cargo/cargo_fun.yml:34]:
```suggestion
    sprite: Objects/Fun/Instruments/harmonica.rsi
```

File: Resources/Prototypes/Catalog/Cargo/cargo_fun.yml [hunk 4]
```diff
@@ -1,23 +1,63 @@
...
+    state: icon
+  product: CrateFunInstrumentsWoodwind
+  cost: 2500
+  category: Fun
+  group: market
+
+- type: cargoProduct
+  id: FunInstrumentsKeyedPercussion
+  icon:
+    sprite: Objects\Fun\Instruments\h_synthesizer.rsi
```
Comment (Endecc) [inline Resources/Prototypes/Catalog/Cargo/cargo_fun.yml:44]:
```suggestion
    sprite: Objects/Fun/Instruments/h_synthesizer.rsi
```

File: Resources/Prototypes/Catalog/Cargo/cargo_fun.yml [hunk 5]
```diff
@@ -1,23 +1,63 @@
...
+    state: icon
+  product: CrateFunInstrumentsKeyedPercussion
+  cost: 2500
+  category: Fun
+  group: market
+
+- type: cargoProduct
+  id: FunInstrumentsSpecial
+  icon:
+    sprite: Objects\Fun\Instruments\gunpet.rsi
```
Comment (Endecc) [inline Resources/Prototypes/Catalog/Cargo/cargo_fun.yml:54]:
```suggestion
    sprite: Objects/Fun/nstruments/gunpet.rsi
```

**Review by Endecc — COMMENTED** [review]
This is what happens when you decide to program something for the first time on a whim. Rookie mistakes have to happen sometimes, I guess!

File: Resources/Prototypes/Catalog/Cargo/cargo_fun.yml [hunk 6]
```diff
@@ -1,23 +1,63 @@
...
+    state: icon
+  product: CrateFunInstrumentsKeyedPercussion
+  cost: 2500
+  category: Fun
+  group: market
+
+- type: cargoProduct
+  id: FunInstrumentsSpecial
+  icon:
+    sprite: Objects/Fun/nstruments/gunpet.rsi
```
Comment (Endecc) [inline Resources/Prototypes/Catalog/Cargo/cargo_fun.yml:54]:
```suggestion
    sprite: Objects/Fun/Instruments/gunpet.rsi
```

**Comment by Emisse** [comment]
lgtm

**Pull request merged**